src/resolvers/auth.js

```diff
@@ -18,17 +18,19 @@ const { CLIENT_URL } = require('../config/config');
 const createResetPasswordEmail = resetToken => {
   const resetPasswordUrl = `${CLIENT_URL}/resetPassword/${resetToken}`;
   return {
-    from: 'kevinzhang@uwblueprint.org',
+    from: 'info@raisingtheroof.org',
     subject: `Raising the Roof Password Reset Attempt`,
-    html: `Reset your password <a href="${resetPasswordUrl}"> here </a>`,
```

Comment:
We should probably add more context to these copies since they're pretty vague and can seem sketchy?

Reply:
yeah, we should create an email template with a footer and all, ugh - gonna track a notion ticket https://www.notion.so/uwblueprintexecs/cc098375fd224e09b6efd05f453b1c5c?v=e4bd24ffa7aa470ba4a7f6b989eae603&p=fb8dee85d3454e24a484a994ab3f5b11